config.json

```diff
@@ -16,6 +16,11 @@
       "label": "CRM getting started project with public apps",
       "path": "projects/public-app-getting-started-template"
     },
+    {
+      "name": "scim-app",
+      "label": "Starter project for integrating with SCIM",
+      "path": "projects/scim-app-template"
+    },
```

Comment:
Sorry, I missed this on the first pass.  We can remove this from here and remove all the changes from the `projects` directory.  The project directory is only used for 2025.1 and previous

Reply:
So, do I put this in the 2025.2 folder?  Otherwise, how do I get the `scim-app-hsmeta.json` created?  I'm trying to make things as simple as I can for the customer, I'd like to be able to say all you need to do is the following to create all the files you need and have as much filled out for them as possible:

```sh
hs init
hs project create --name my-scim-project --template scim-app
```